sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -286,6 +288,11 @@ static void applyMetadata(
           options.setTracePropagationTargets(tracePropagationTargets);
         }
 
+        if (metadata.containsKey(PERFORM_ACTIVITY_FRAMES_TRACKING)) {
```

Comment:
I think we don't need this check, because `readBool` does it already and fallbacks to default value in case it's not in the manifest